pt_pud-ud-test.conllu

```diff
@@ -157,15 +157,15 @@
 13	mas	_	CCONJ	CC	_	19	cc	_	_
 14	o	_	DET	DT	Gender=Masc|Number=Sing	15	det	_	_
 15	destino	_	NOUN	NN	Gender=Masc|Number=Sing	19	nsubj	_	_
-16-17	dos	_	_	_	_	_	_	_	_
+16-17	da	_	_	_	_	_	_	_	_
 16	de	de	ADP	INDT	_	18	case	_	_
-17	os	o	DET	_	Gender=Masc|Number=Plur	18	det	_	_
-18	republicanos	_	NOUN	NN	Gender=Masc|Number=Plur	15	nmod	_	_
+17	a	o	DET	_	Gender=Fem|Number=Plur	18	det	_	_
```

Comment:
Shouldn't _a_ be `Number=Sing`?

Reply:
solved. it is always good to have a second person double-checking.